app/cafe.py

```diff
@@ -0,0 +1,20 @@
+from datetime import date
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("NotVaccinatedError")
+        elif visitor["vaccine"]["expiration_date"] < date.today():
+            raise OutdatedVaccineError("OutdatedVaccineError")
+        elif not visitor["wearing_a_mask"]:
+            raise NotWearingMaskError("NotWearingMaskError")
```

Comment:
Add descriptive error message; 
I mean may be use visitor name for better error message : )